t/e2e/whisper/whisper_test.go

```diff
@@ -92,26 +92,3 @@ func (s *WhisperTestSuite) TestWhisperFilterRace() {
 
 	<-allFiltersAdded
 }
-
-func (s *WhisperTestSuite) TestLogout() {
```

Comment:
I think we could keep this test just use `StatusBackend.Logout()` to make sure that Whisper keys are removed.

Reply:
Yep, I should move it to a different test suite as this one is based on `e2e.NodeManagerTestSuite`.

I guess the correct one could be `AccountsTestSuite`. But there is a [test already checking the same behaviour](https://github.com/status-im/status-go/pull/753/files#diff-ccd7ed05a6d1758583d789680e76de2bR291) there.

Reply:
I think it's fine because `Backend.Logout()` still depends on `NodeManager`. What's the name of that test? I can't find it following the link.

Reply:
hmm, try this one : [AccountsTestSuite/TestSelectedAccountOnRestart](https://github.com/status-im/status-go/blob/2d964bfe9f9554fdb34c6c19095f3c5e88c527e6/t/e2e/accounts/accounts_test.go#L219)

Reply:
Ah, I see. Hm, so I would actually split that test so in `t/e2e/accounts/accounts_test.go` we check only stuff related to `accounts.Manager` and as Whisper is not present `geth/account` anymore, I would move this check maybe here, to `t/e2e/whisper/`.

Reply:
Sounds reasonable. 

Some changes need to happen on `t/e2e/whisper/whisper_test.go` actually it does not have access to `StatusBackend` so it will need to implement `e2e.BackendTestSuite` instead of `e2e.NodeManagerTestSuite`.

Do you think it makes sense to create an issue to tackle this test splitting instead of keep making big this PR?

Reply:
New issue created to solve this: https://github.com/status-im/status-go/pull/753